Clients/GUI/wwwroot/css/Style.css

```diff
@@ -81,4 +81,64 @@
 
     .cur-icon .gbp-icon {
         background-image: url(/images/currencies/gbp.png)
-    }
\ No newline at end of file
+    }
+
+body {
```

Comment:
Why do you need to add new styles? Can you use bootstrap?

Reply:
I found this ready-made code earlier than something similar in bootstrap

Reply:
> Why do you need to add new styles? Can you use bootstrap?

May be you are right, but Dima's styles look good so we can use them and not spend time on bootstrap in this task due to lack of time